central-pet-front/src/index.css

```
@@ -1 +1,97 @@
 @import 'tailwindcss';
+
+@theme {
```

Stylelint (17.7.0)

[error] Unexpected unknown at-rule "@theme" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)